core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -415,6 +421,8 @@ def __extend_domain(self, domain_home):
             self.wlst_helper.update_domain()
         elif self._domain_typedef.is_jrf_domain_type():
             self.target_helper.target_jrf_groups_to_clusters_servers(domain_home)
+        else:
```

Comment:
Why do you bypass setServerGroups when you have a custom template?

Reply:
Not planning to implement custom domain (non-extension) templates for this issue.